src/stores/widgets/SpectralProfileWidgetStore.ts

```diff
@@ -40,16 +60,16 @@ export class SpectralProfileWidgetStore extends RegionWidgetStore {
         }
     }
 
-    // Qi, add new stoke valid type PI, PA, Qz+Uz, QzVsUz
-    private static ValidCoordinates = ["z", "Iz", "Qz", "Uz", "Vz", "PIz", "PAz"];
+    // add new stoke valid type PI, PA, Qz+Uz, QzVsUz
+    private static ValidCoordinates = ["z", "Iz", "Qz", "Uz", "Vz", "PIz", "PAz", "Qz+Uz"];
```

Comment:
These could also use the enum (and you can type the array as `: Array<StokesCoordinate>` for consistency)